picard/ui/options/general.py

```diff
@@ -61,12 +65,18 @@ def load(self):
         self.ui.server_port.setValue(config.setting["server_port"])
         self.ui.analyze_new_files.setChecked(config.setting["analyze_new_files"])
         self.ui.ignore_file_mbids.setChecked(config.setting["ignore_file_mbids"])
+        self.ui.check_for_updates.setChecked(config.setting["check_for_updates"])
+        self.ui.update_level.setCurrentIndex(config.setting["update_level"])
+        self.ui.update_check_days.setValue(config.setting["update_check_days"])
 
     def save(self):
         config.setting["server_host"] = self.ui.server_host.currentText().strip()
         config.setting["server_port"] = self.ui.server_port.value()
         config.setting["analyze_new_files"] = self.ui.analyze_new_files.isChecked()
         config.setting["ignore_file_mbids"] = self.ui.ignore_file_mbids.isChecked()
+        config.setting["check_for_updates"] = self.ui.check_for_updates.isChecked()
+        config.setting["update_level"] = self.ui.update_level.currentIndex()
```

Comment:
It is a bad idea to use the list index, because if we re-order the list it will change the meaning of the option, you need to have some indirection here: index -> value, and store that value as option value. `setData()` and UserRole would work (but you can't set those in qt designer).

Reply:
I reworked this a lot and now refer everything back to a constant dictionary defined in the picard `__init__.py` file.  The selection information is now taken from `self.ui.update_level.currentData(QtCore.Qt.UserRole)`.